common/types/keys.go

```diff
@@ -31,7 +31,7 @@ const (
 )
 
 const (
-	TimerVersionKey = "Entry_Version"
```

Comment:
this needs migration?

Reply:
can't do migration, because the timer version is mixed up with that of the fixation store that owns it (if any). Instead, will add logic on the next getVersion() call, to test and if not initialize then initialize ad-hoc. the call to getVersion() would happen naturally upon next upgrade as the logic needs to decided whether migration is needed at all.